x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -70,70 +79,156 @@ export function LayerContextMenu(props: VisualizationLayerWidgetProps<State>) {
 
 const idPrefix = htmlIdGenerator()();
 
-export function DimensionEditor({
-  state,
-  setState,
-  layerId,
-  accessor,
-}: VisualizationDimensionEditorProps<State>) {
+export function DimensionEditor(props: VisualizationDimensionEditorProps<State>) {
+  const { state, setState, layerId, accessor } = props;
   const index = state.layers.findIndex((l) => l.layerId === layerId);
   const layer = state.layers[index];
   const axisMode =
     (layer.yConfig &&
       layer.yConfig?.find((yAxisConfig) => yAxisConfig.forAccessor === accessor)?.axisMode) ||
     'auto';
+
   return (
-    <EuiFormRow
-      display="columnCompressed"
-      label={i18n.translate('xpack.lens.xyChart.axisSide.label', {
-        defaultMessage: 'Axis side',
-      })}
-    >
-      <EuiButtonGroup
-        legend={i18n.translate('xpack.lens.xyChart.axisSide.label', {
+    <EuiForm>
+      <EuiFormRow
+        display="columnCompressed"
```

Comment:
Add `fullWidth` on all the form rows

```suggestion
        display="columnCompressed"
        fullWidth
```